L10nSnapshotTests/L10nMktSuiteSnapshotTests.swift

```diff
@@ -0,0 +1,164 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* Disabled by default, running locally to get these specific screenshots in specific locales
+import XCTest
+
+class L10nMktSuiteSnapshotTests: L10nBaseSnapshotTests {
+
+    func test1SettingsETP() {
+        navigator.goto(TrackingProtectionSettings)
+        
+       // Check the warning alert
+        app.cells["Settings.TrackingProtectionOption.BlockListStrict"].tap()
+        app.alerts.buttons.firstMatch.tap()
+        sleep(3)
+        snapshot("TrackingProtectionStrictWarning-01")
+        waitForExistence(app.cells["Settings.TrackingProtectionOption.BlockListBasic"])
+    }
+    
+    func test2URLBar() {
+        navigator.openURL("https://www.kingarthurbaking.com/recipes/sourdough-starter-recipe")
```

Comment:
Is it possible to use Mozilla sites in this test or are these the sites marketing requested?

Reply:
I used these as I was trying to get same screenshot mkt gave as an example. We can change that if they prefer to use others...